src/cmd/initialize.go

```diff
@@ -52,73 +52,81 @@ var initCmd = &cobra.Command{
 		}
 
 		// Continue running package deploy for all components like any other package
-		initPackageName := fmt.Sprintf("zarf-init-%s.tar.zst", config.GetArch())
+		initPackageName := config.GetInitPackageName()
 		config.DeployOptions.PackagePath = initPackageName
 
 		// Try to use an init-package in the executable directory if none exist in current working directory
 		if utils.InvalidPath(config.DeployOptions.PackagePath) {
+			// Get the path to the executable
 			executablePath, err := utils.GetFinalExecutablePath()
-			if err != nil {
```

Comment:
I know this is to combine the check below but I would still recommend checking errors sooner rather than later. It would be pretty simple for someone to come modify this code in the future and add some logic in-between these two sections not realizing that the error checking was being delayed.

Reply:
There are no error generators between it's declaration and usage, but you make a good point.  I'll go ahead and log the error too.